docs/6.x/monitoring.md

```diff
@@ -1,46 +1,67 @@
-are generated during initial installation and placed into a secret `grafana` in `monitoring` namespace.
+[Grafana](https://grafana.com/) is an open-source metrics analytics and visualization suite.
+
+In Gravity clusters Grafana uses Prometheus as a data-source. It is preconfigured with several
+dashboards that provide general information about individual nodes, containers and the overall
+cluster health.
+
+!!! tip:
+    When building a cluster image, it is possible to add your own dashboards in addition to
+    the ones that ship by default. See [Grafana Integration](#grafana-integration) below for
+    details.
+
+Grafana exposes the cluster-only service `grafana.monitoring.svc.cluster.local:3000`.
+
+### Alertmanager
+
+[Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) is a Prometheus component that
+handles alerts sent by Prometheus server and takes care of deduplicating, grouping and routing
+them to the correct receiver such as email recipient.
 
-### Kapacitor
+Alertmanager exposes the cluster-only service `alertmanager-main.monitoring.svc.cluster.local:9093`.
 
-Kapacitor is the alerting system that streams data from InfluxDB and sends alerts as configured by
-the end user. It exposes the service `kapacitor.monitoring.svc.cluster.local`.
+## Grafana Integration
 
-## Grafana integration
+The default Grafana configuration includes two pre-configured dashboards providing machine- and
+pod-level overview of the installed cluster by default. Grafana UI is integrated with Gravity
+control panel. To view dashboards once the cluster is up and running, navigate to the Cluster's
+Monitoring page.
 
-The standard Grafana configuration includes two pre-configured dashboards providing machine- and pod-level overview
-of the installed cluster by default. The Grafana UI is integrated with Gravity control panel. To view dashboards once
-the cluster is up and running, navigate to the Cluster's Monitoring page.
+By default, Grafana is running in anonymous read-only mode which allows anyone logged into Gravity
+to view existing dashboards (but not modify them or create new ones).
 
-By default, Grafana is running in anonymous read-only mode which allows anyone logged into Gravity to view existing
-dashboards (but not modify them or create new ones).
+## Pluggable Dashboards
 
-## Pluggable dashboards
+Your applications can create their own Grafana dashboards using ConfigMaps.
 
-Your applications can use their own Grafana dashboards using ConfigMaps.
-A custom dashboard ConfigMap should be assigned a `monitoring` label with value `dashboard` and created in the
-`monitoring` namespace so it is recognized and loaded when installing the application:
+A custom dashboard ConfigMap should be assigned a `monitoring` label with value `dashboard` and
+created in the `monitoring` namespace so it is recognized and loaded during initial cluster image
+installation:
```

Comment:
A custom dashboard ConfigMap needs to be placed into the `monitoring` namespace and assigned the special `monitoring:dashboard` label so it is recognized as a dashboard.